pensjon-brevbaker-api-model/src/main/kotlin/no/nav/pensjon/brev/api/model/vedlegg/OpplysningerOmAvdodBruktIBeregning.kt

```diff
@@ -0,0 +1,115 @@
+    val avdoedTrygdetidAvtaleland: List<Trygdetid>,
+    val avdoedPoengrekkeVedVirk: AvdoedPoengrekkeVedVirk
+) {
+    data class AvdoedPoengrekkeVedVirk(
+        val inneholderFramtidigPoeng: Boolean,
+        val inneholderOmsorgspoeng: Boolean,
+        val pensjonspoeng: List<Pensjonspoeng>,
+    )
+    data class AvdodBeregningKap3(
+        val sluttpoengtall: Double?,
+        val sluttpoengtallMedOverkomp: Double?,
+        val sluttpoengtallUtenOverkomp: Double?,
+        val poengAr: Int?,
+        val poengAre91: Int?,
+        val poengArf92: Int?,
+        val poengArNevner: Int?,
+        val poengArTeller: Int?,
+        val framtidigPoengAr: Int?,
+    )
+    data class AvdoedYrkesskadedetaljerVedVirk(
+        val sluttpoengtall: Double?,
+        val poengAr: Int?,
+        val poengAre91: Int?,
+        val poengArf92: Int?,
+        val yrkesskadeUforegrad: Int?,
+    )
+
+    data class AvdoedBeregningKap19VedVirk(
+        val faktiskPoengArAvtale: Int?,
+        val faktiskPoengArNorge: Int?,
+        val framtidigPoengAr: Int?,
+        val poengAr: Int?,
+        val poengArNevner: Int?,
+        val poengArTeller: Int?,
+        val poengAre91: Int?,
+        val poengArf92: Int?,
+        val sluttpoengtall: Double?,
+        val sluttpoengtallMedOverkomp: Double?,
+        val sluttpoengtallUtenOverkomp: Double?,
+    )
+
+    data class TilleggspensjonVedVirk(
+        val kombinertMedAvdoed: Boolean,
+        val pgaUngUforeAvdod: Boolean,
+    )
+
+    data class AvdoedTrygdetidsdetaljerVedVirkNokkelInfo(
+        val beregningsMetode: Beregningsmetode,
+        val framtidigTTEOS: Int?,
+        val framtidigTTBilateral: Int?,
+        val anvendtTT: Int?,
+        val faktiskTTNordiskKonv: Int?,
+        val framtidigTTNorsk: Int?,
+        val tellerTTEOS: Int?,
+        val nevnerTTEOS: Int?,
+        val tellerProRata: Int?,
+        val nevnerProRata: Int?,
+    )
+
+    data class AlderspensjonVedVirk(
+        val regelverkType: AlderspensjonRegelverkType,
+        val gjenlevenderettAnvendt: Boolean,
+        val tilleggspensjonInnvilget: Boolean,
+    )
+
+    data class AvdoedTrygdetidsdetaljerKap19VedVirk(
+        val anvendtTT: Int?,
+        val beregningsMetode: Beregningsmetode,
+        val framtidigTTNorsk: Int?,
+        val faktiskTTNordiskKonv: Int?,
+        val tellerTTEOS: Int?,
+        val nevnerTTEOS: Int?,
+        val framtidigTTEOS: Int?,
+        val framtidigTTBilateral: Int?,
+        val tellerProRata: Int?,
+        val nevnerProRata: Int?,
+    )
+
+    data class Avdoed(
+        val navn: String,
+        val avdoedFnr: Foedselsnummer,
+    )
+
+    data class Bruker(
+        val foedselsdato: LocalDate,
+    )
+
+    data class BeregnetPensjonPerManedVedVirk(
+        val virkDatoFom: LocalDate,
+        val avdoedFlyktningstatusErBrukt: Boolean,
+    )
+
+    data class Foedselsnummer(override val value: String) : no.nav.pensjon.brevbaker.api.model.Foedselsnummer
```

Comment:
Vinn vi noko på å ha denne kontra å bruke den eksisterande direkte?

Reply:
Å ikke bruke FoedselsnummerImpl tenker du? Så den var markert som intern.

Reply:
Ser ikke helt for meg at hvordan vi lagrer/viser fødselsnummer endrer seg noe særlig. Om vi skal gjøre det pent og pyntlig bør kanskje DSL/formattering eie disse value wrapperne på noe vis slik at mal-rendring og maler bare må tilpasse seg hva nå enn format krever.

Reply:
Stemmer egentlig for å bare fjerne interface, impl og annotasjon.

Reply:
I andre iterasjon (brukt i brevbaker-dsl-modulen) endte vi på å heller enn å gå til interface+impl bruke vanleg class i staden for data class, pluss internal constructor. Tenkjer vi bør gå over til det også der vi har brukt interface+impl.

Reply:
Men akkurat her og no treng du jo ikkje impl, bruk Foedselsnummer-interfacet? Vi generer vel selectors også for interfaces.